src/func.py

```diff
@@ -3,4 +3,9 @@
 # Read a number from the keyboard
 num = input("Enter a number: ")
 
-# Print out "Even!" if the number is even. Otherwise print "Odd"
\ No newline at end of file
+# Print out "Even!" if the number is even. Otherwise print "Odd"
+
+if int(num) % 2 == 0:
+    print(True)
```

Comment:
Remember to follow the specifications carefully -- this should print out "Even" or "Odd", not True/False :)

Reply:
Oh wow! I can't believe I did that haha.